modules/sso.php

```diff
@@ -220,8 +220,16 @@ public function login_body_class( $classes ) {
 		//Always add the jetpack-sso class so that we can add SSO specific styling even when the SSO form isn't being displayed.
 		$classes[] = 'jetpack-sso';
 
-		// If jetpack-sso-default-form, show the default login form.
-		if ( isset( $_GET['jetpack-sso-default-form'] ) && 1 == $_GET['jetpack-sso-default-form'] ) {
+		/**
+		 * Should we show the default or SSO login form?
+		 *
+		 * $_GET['jetpack-sso-default-form'] is used to provide a fallback in case JavaScript is not enabled.
+		 *
+		 * The default_to_sso_login() method allows us to dynamically decide which type of login to show. The SSO module
+		 * uses the method to display the default login form if we can not find a user to log in via SSO. But, the method
+		 * could be filtered by a site admin to always show the default login form if that is preferred.
+		 */
+		if ( ! empty( $_GET['jetpack-sso-default-form'] )  || ! Jetpack_SSO_Helpers::default_to_sso_login() ) {
```

Comment:
Thanks for the feedback.

I like your option.

Or, perhaps we just updated the first part of the comment to be:

```
Should we show the default login form?
```